### DISCUSSION TITLE
Using second device on USB sound card

### DISCUSSION THREAD

**Discussion by sdhannay** [rern/rAudio#1744] [General]
Hi, I have a preamp (Emotiva PT-100) with a built in USB DAC.

It displays in 'playback' configuration, however I has issues playing anything but 44.1/16

Some investigation it seems the device works if I select hw:2,1 not hw:2,0

Is there a way to have rAudio select the select the second device on the card rather than the default 0  ?

I made some manual tweaks to mpd.conf and shairport-sync.conf confirmed hw:2,1 works, however it does not stick after a reboot.

Anybody have an idea on the best way to handle this arguably non-standard DAC ?

Attached is the alsa-info output.

[alsa-info.txt](https://github.com/user-attachments/files/16715793/alsa-info.txt)

Thanks

**Comment by kan-ibal** [comment]
As far as I understand your issue, you can set device output in mpd custom config. Settings->Player->Option->User's Configuration and set mpd's  audio_output section rule 'device                  "hw:2,1".

**Reply by sdhannay** [reply]
Thanks.

Yes modifying the mpd config works (has to be done manually, as the custom config does not impact he auto generated mpd output configuration) , however it does not carry through to shairport config etc.

**Comment by sdhannay** [comment]
Sorry about the red herring here. Putting up the discovery in case somebody else comes across this anomaly.

Turns out that the behavior of this particular DAC is somehow related to the raspberry pi4. I used the same SDcard (no config changes) in a Raspberry pi3 and there were no issues at all. Able to use the stock rAudio with zero issues.
